hack/after-build/update-generated-protobuf.sh

```diff
@@ -21,7 +21,10 @@ set -o pipefail
 KUBE_ROOT=$(dirname "${BASH_SOURCE}")/../..
 source "${KUBE_ROOT}/hack/lib/init.sh"
 
-# TODO: preinstall protobuf
+kube::golang::setup_env
+
+hack/build-go.sh cmd/libs/go2idl/go-to-protobuf cmd/libs/go2idl/go-to-protobuf/protoc-gen-gogo
```

Comment:
This is not exactly how I would like this to look like, but when I build those binaries in the:
hack/update-generated-protobuf.sh
gotoprotobuf just doesn't work - I'm getting errors like these:

    type checker error: k8s.io/kubernetes/pkg/api/resource/quantity.go:20:2: could not import errors (unable to *find*     "errors": cannot find package "errors" in any of:
    	/usr/lib/google-golang/src/errors (from $GOROOT)
	    /go/src/k8s.io/kubernetes/_output/local/go/src/errors (from $GOPATH)
    	/go/src/k8s.io/kubernetes/Godeps/_workspace/src/errors)

If you have any suggestions how to fix it - I'm happy to change it.